File: test/ocamlTests/BoolTest/fromStringTest.ml
test "fromString("true")" (fun () -> expect (Bool.fromString "true") |> toEqual (let open Eq in option bool) Some(true)) ; 
test "fromString("false")" (fun () -> expect (Bool.fromString "false") |> toEqual (let open Eq in option bool) Some(false)) ; 
test "fromString("True")" (fun () -> expect (Bool.fromString "True") |> toEqual (let open Eq in option bool) None) ; 
test "fromString("1")" (fun () -> expect (Bool.fromString "1") |> toEqual (let open Eq in option bool) None) ; 

File: test/fsharpTests/BoolTest/fromStringTest.fs
testCase "fromString("true")" 
<| fun _ -> 
    let expected = Some(true)
    Expect.equal expected (Bool.fromString "true") "error"
testCase "fromString("false")" 
<| fun _ -> 
    let expected = Some(false)
    Expect.equal expected (Bool.fromString "false") "error"
testCase "fromString("True")" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Bool.fromString "True") "error"
testCase "fromString("1")" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Bool.fromString "1") "error"
